packages/edge-demo-site/remix.config.js

```diff
@@ -0,0 +1,13 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { config } = require('@netlify/remix-edge-adapter')
```

Comment:
This is the new way of providing sensible defaults for Netlify support of Remix. If someone wants to override/enhance them, they can.